pkg/graph/schema.resolvers.go

```diff
@@ -2056,7 +2056,7 @@ func (r *tRBRequestResolver) RequesterInfo(ctx context.Context, obj *models.TRBR
 // RequesterComponent is the resolver for the requesterComponent field.
 func (r *tRBRequestResolver) RequesterComponent(ctx context.Context, obj *models.TRBRequest) (*string, error) {
 	requester := obj.CreatedBy
-	return resolvers.GetTRBUserComponent(ctx, r.store, &requester, obj.ID)
+	return resolvers.GetTRBAttendeeComponent(ctx, r.store, &requester, obj.ID)
```

Comment:
Here's the reason for the rename. The `RequesterComponent` resolver passes in the requester EUA, but the store/resolver methods aren't restricted to just the requester and can be used for any attendee EUA (if we ever needed to do that).